src/components/card/card.scss

```diff
@@ -0,0 +1,211 @@
+
```

Comment:
Couple of concerns about the css overall:
* In places for you use margin and padding left/right, I think these will be off for RTL layouts.
* A lot of the sizing properties are coded directly into the styles, which would make it hard to customize for compactness. Would there be a straightforward way of extracting the sizing information into variables?